datareduction/datareducer.cpp

```diff
@@ -1196,13 +1196,16 @@ bool DataReducer::writeFsGridData(
                       FsGrid< std::array<Real, fsgrids::dmoments::N_DMOMENTS>, 2>& dMomentsGrid,
                       FsGrid< std::array<Real, fsgrids::bgbfield::N_BGB>, 2>& BgBGrid,
                       FsGrid< std::array<Real, fsgrids::volfields::N_VOL>, 2>& volGrid,
-                      FsGrid< fsgrids::technical, 2>& technicalGrid, const std::string& meshName, const unsigned int operatorID, vlsv::Writer& vlsvWriter) {
+                      FsGrid< fsgrids::technical, 2>& technicalGrid, const
+                      std::string& meshName, const unsigned int operatorID,
+                      vlsv::Writer& vlsvWriter,
```

Comment:
Interesting line break pattern...

Reply:
Whoopsie. I screwed that up consistently it seems. Fixed.